libs/components/modals/src/lib/modules/modal/modal-footer.component.scss

```diff
@@ -16,6 +16,11 @@
   .sky-btn + .sky-btn-link {
     margin-left: -2px;
   }
+
+  /* Should this be all status indicators or just ours? */
+  .sky-modal-footer-errors .sky-status-indicator {
+    margin-bottom: var(--sky-margin-stacked-lg);
```

Comment:
Just confirming: Adam recommended this value for the margin?

Reply:
Yes, Adam and Paul suggested we use this variable